homeassistant/components/zwave/__init__.py

```diff
@@ -542,6 +546,12 @@ def print_config_parameter(service):
         _LOGGER.info("Config parameter %s on Node %s : %s",
                      param, node_id, get_config_value(node, param))
 
+    def print_node(service):
+        """Print all information about z-wave node."""
+        node_id = service.data.get(const.ATTR_NODE_ID)
+        node = NETWORK.nodes[node_id]
+        nice_print_node(node)
```

Comment:
Is it intentional to use the console for output? This will then only be visible in the console output or journalctl, and not in the log file by default.
If debug is enabled this will be done for all nodes, and is it of any use to the normal user to get all the values of the node?